apps/browser/src/fido2/popup/fido2/fido2.component.html

```diff
@@ -0,0 +1,58 @@
+<!-- TODO: Rewrite and refactor this component when implementing the new design -->
+<ng-container *ngIf="data$ | async as data">
+  <div class="auth-wrapper">
+    <i class="bwi bwi-spinner bwi-lg bwi-spin" [hidden]="!loading" aria-hidden="true"></i>
+    <ng-container *ngIf="data.showUnsupportedVerification">
+      Verification required by the initiating site. This feature is not yet implemented for accounts
+      without master password.
+    </ng-container>
+    <ng-container *ngIf="!data.showUnsupportedVerification">
+      <ng-container
+        *ngIf="
+          data.message.type == 'PickCredentialRequest' ||
+          data.message.type == 'ConfirmNewNonDiscoverableCredentialRequest'
+        "
+      >
+        A site is asking for authentication, please choose one of the following credentials to use:
+        <div class="box list">
+          <div class="box-content">
+            <app-cipher-row
+              *ngFor="let cipher of ciphers"
+              [cipher]="cipher"
+              (onSelected)="pick(cipher)"
+            ></app-cipher-row>
+          </div>
+        </div>
+      </ng-container>
+      <ng-container *ngIf="data.message.type == 'ConfirmNewCredentialRequest'">
+        A site wants to create the following passkey in your vault
+        <div class="box list">
+          <div class="box-content">
+            <app-cipher-row [cipher]="ciphers[0]"></app-cipher-row>
+          </div>
+        </div>
+        <button type="button" class="btn btn-outline-secondary" (click)="confirm()">Create</button>
+      </ng-container>
+      <ng-container *ngIf="data.message.type == 'InformExcludedCredentialRequest'">
+        A passkey already exists in Bitwarden for this account
+        <div class="box list">
+          <div class="box-content">
+            <app-cipher-row *ngFor="let cipher of ciphers" [cipher]="cipher"></app-cipher-row>
+          </div>
+        </div>
+      </ng-container>
+      <ng-container *ngIf="data.message.type == 'InformCredentialNotFoundRequest'">
+        You do not have a matching login for this site.
+      </ng-container>
+    </ng-container>
+    <button
+      *ngIf="data.fallbackSupported"
+      type="button"
+      class="btn btn-outline-secondary"
+      (click)="abort(true)"
+    >
```

Comment:
```suggestion
    <button
      *ngIf="data.fallbackSupported"
      class="btn btn-outline-secondary"
      (click)="abort(true)"
    >
```
Adding `type="button"` is not necessary here.

Reply:
I'm incorrect about the above statement, resolving this comment.